ReadMe.md

```diff
@@ -36,20 +36,20 @@ If you're interested, why not start with:
 ### Motivation
 
 - Automate the process which needs to be done regularly.
-- Make the task easy and stress free of managment members.
-- Save the time of people in doing same work multiple times.
+- Make the task easy and stress-free for management members.
+- Save the time of people in doing the same work multiple times.
 
 ### Methodology
 
-We are following **Spotify Agile Methodology** to give the contributors an idea of efficient software development methods and how work is co-ordinated in the industry. If you don't know what that means, please refer [this blog](https://blog.praveen.science/my-experience-on-spotify-agile-methodology/). 
+We are following **Spotify Agile Methodology** to give the contributors an idea of efficient software development methods and how work is coordinated in the industry. If you don't know what that means, please refer to [this blog](https://blog.praveen.science/my-experience-on-spotify-agile-methodology/). 
 
 Following are some points on the methodology to be used:
 
 * To get the best out of everyone contributing to this project, we have adopted _high alignment and high autonomy model_.
 * The issues created will have a description of the problem to be solved, the contributors have to figure out how they would like to solve it and what tool they would like to use.
-* Everything needs to be communicated to the scrum masters / project admin on what they think about a particular issue and its possible solution.
-* We will have a sprint of one variable days week, based on the level of difficulty of the issue with regular sprint planning sessions and daily stand-ups to track the progress of each and every contributor.
-* In these sessions, the scrum masters will take a note of the progress and blockers if any. If the contributor faces any issues / blocks, the others will help them solve it and resume their work. Hence, the blocks can be addressed timely.
+* Everything needs to be communicated to the scrum masters/project admin on what they think about a particular issue and its possible solution.
+* We will have a sprint of one variable day a week, based on the level of difficulty of the issue with regular sprint planning sessions and daily stand-ups to track each contributor’s progress.
+* In these sessions, the scrum masters will take note of the progress and blockers if any. If the contributor faces any issues/blocks, the others will help them solve them and resume their work. Hence, the blocks can be addressed timely.
```

Comment:
Let's please not remove the spaces around `/`.
```suggestion
* In these sessions, the scrum masters will take note of the progress and blockers if any. If the contributor faces any issues / blocks, the others will help them solve them and resume their work. Hence, the blocks can be addressed timely.
```

Reply:
okay sir